lib/driver.js

```diff
@@ -1025,8 +1054,7 @@ export class XCUITestDriver extends BaseDriver {
         await this.wda.quit();
       }
     }
-
-    DEVICE_CONNECTIONS_FACTORY.releaseConnection(this.opts.udid);
+    DEVICE_CONNECTIONS_FACTORY.releaseConnection(this.opts.udid, this.opts.mjpegServerPort);
```

Comment:
please keep it as is as not providing this.opts.mjpegServerPort means to release connections on all ports for the given device udid